docs/all-views-inventory.md

```diff
@@ -0,0 +1,437 @@
+- [CIIP Application Disclaimer `/resources/application-disclaimer`](#application-disclaimer)
+- ['Page Not Found' page (_aka._ 404)](#page-404)
+- [403 page (temporary; handles redirection after registration bug) `/403`](#page-403)
+- [Unknown Error page, _aka._ ErrorBoundary `(any URL)`](#error-boundary)
+- [BCGov-wide Website Disclaimer `/resources/disclaimer`](#website-disclaimer)
+- [BCGov-wide Privacy Statement `/resources/privacy`](#privacy)
+- [BCGov-wide Copyright Statement `/resources/copyright`](#copyright)
+- [Contact page `/resources/contact`](#contact)
+
+### [Note on required query parameters](#query-parameters)
+
+### [Globally shared components](#global-components)
+
+_______
+
+<a id="industry-user-role"></a>
+
+## Industry User
+
+Industry users (reporters and certifiers) must log in using an email address and are not expected to have an IDIR.
+
+<a id="reporter-dash"></a>
+
+### Reporter Dashboard: `/reporter`
+
+View and request access to operators to apply for CIIP on behalf of.
+
+#### Shared components:
+- `Organisation` (dropdown item in organisation search)
+
+<a id="facilities"></a>
+
+### Facilities page, _aka._ "My Applications": `/reporter/facilities`
+
+View facilities for operators to which the user has access and on behalf of which they can apply for CIIP. The listings provide links to the following actions for each facility:
+
+  - a) start an application
+  - b) resume an in-progress application
+  - c) view a submitted application
+
+**Note:** When navigating to this page via "My Applications", all applications from all the user's facilities and operators are listed. Otherwise when navigating here by clicking on an individual operator, this page shows just the applications for the facilities belonging to that operator.
+
+#### Shared components:
+- `SearchTable`
+
+<a id="new-application-disclaimer"></a>
+
+### New Application Disclaimer page: `/reporter/new-application-disclaimer`\*
+
+Interstitial page that collects the reporter's consent before allowing a new application to be started.
+
+\* Required query parameters:
+- `applicationId`
+- `version`
+
+<a id="application"></a>
+
+### Application page: `/reporter/application`\*
+
+A set of forms, displayed in sequential tabs, allowing reporters to apply for CIIP for a specified facility.
+
+The form tab (ie: Emissions, etc.) initially displayed on loading this page is controlled by the value of the `formResultId` query parameter. Further, the page can be loaded to display the Summary tab by appending the parameter: `confirmationPage=true`.
+
+Form validation helps ensure the integrity of data entered before submission; however, this can be overridden by the reporter with a message justifying their reasons for not being able to fix the validation errors.
+
+The current state of the application can be downloaded as PDF.
+
+\* Required query parameters:
+- `applicationId`
+- `version`
+- `formResultId`
```

Comment:
You're right - no, and in fact it's confusing if included when `confirmationPage` is present, because then the Summary tab is shown but the tab corresponding to that other `formResultId` is also highlighted. 

I say when `confirmationPage` is present because it actually doesn't matter if you set it to true, false or any other value. The summary tab is shown based on the presence/absence of that parameter.

🙄 It will be nice to replace this quirky behaviour with dynamic route parameters next version of CIIP!

Have updated the description accordingly in a way that I hope doesn't add too much confusing info.